Lombiq.Tests.UI/Extensions/VisualVerificationUITestContextExtensions.cs

```diff
@@ -400,22 +401,29 @@ public static class VisualVerificationUITestContextExtensions
         // We take a screenshot of the element area. This will be compared to a baseline image.
         using var elementImageOriginal = context.TakeElementScreenshot(element).ShouldNotBeNull();
```

Comment:
We take element screenshot here, and again when calling:
https://github.com/Lombiq/UI-Testing-Toolbox/blob/b43d44ed235e65a487e2f66053f91e17e00cad9c/Lombiq.Tests.UI/Extensions/VisualVerificationUITestContextExtensions.cs#L422

Can't we reuse the existing one?

Reply:
Now it is reused.